convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -88,7 +94,13 @@ def run(self):
         super(SubscribeSystem, self).run()
 
         if toolopts.tool_opts.no_rhsm:
-            logger.warning("Detected --no-rhsm option. Skipping.")
+            message = "Detected --no-rhsm option. Skipping."
+            logger.warning(message)
+            self.add_message(
+                level="WARNING",
+                id="NO_RHSM_SKIP",
```

Comment:
@abadger, what's the main purpose of the `id`? Shouldn't it be unique to make it possible to aggregate results from multiple machines? To make it possible to say: _"This one message was hit on machines x,y,x."_

Reply:
I think it would be a combination of action_id + id, in which the action_id is unique across every action and the id itself could be repeated, as for example; 

```
PreSubscription.NO_RHSM_SKIP
DbusCheck.NO_RHSM_KIP
SomeOtherCheck.UNKNOWN_ERROR
AVeryImportantCheck.UNKNOWN_ERROR
```

If I remember correctly, we are using UNKNOWN_ERROR now to say that something happened during the execution of the action, and we didn't expect that to happen (mainly logger.critical)